debian/clearwater-etcd.init.d

```diff
@@ -134,7 +134,10 @@ join_cluster()
           then
             echo "Failed to remove local node from cluster"
             exit 2
-          fi
+          elif [[ -d $DATA_DIR/$advertisement_ip ]]
+          then 
+            rm -rf $DATA_DIR/$advertisement_ip
```

Comment:
Please no `-f` especially when the only non-variable character in the path is `/`.